logstash-core/lib/logstash/pipeline.rb

```diff
@@ -38,6 +38,7 @@ def self.validate_config(config_str, settings = {})
     begin
       # There should be a better way to test this
       self.new(config_str, settings)
+      true
     rescue => e
       e.message
```

Comment:
I think I would go for a more uniform interface for this method, also might be good to add some documentation so we don't make mistakes with return expectations.